doc/cephadm/services/index.rst

```diff
@@ -81,6 +81,13 @@ system name:
 
     ceph orch ps --daemon_type osd --daemon_id 0
 
+.. note::
+   The information shown by ceph orch ps may not be up to date. Daemon status is refreshed by default each 10min.
```

Comment:
Suggest double backticks \`\`ceph orch ps\`\`
"Daemon status is refreshed every 10 minutes by default."

Reply:
Fixed